includes/Actions/ZohoCRM/ZohoCRMController.php

```diff
@@ -640,6 +640,25 @@ public function execute($integrationData, $fieldValues)
         return $zcrmApiResponse;
     }
 
+    public static function getUserDetails($requestParams)
+    {
+        if (empty($requestParams->code) || $requestParams->code === null) {
+            return;
+        }
+        $apiEndpoint = 'https://www.zohoapis.com/crm/v2/users?type=AllUsers';
+        $authorizationHeader['Authorization'] = 'Bearer ' . $requestParams->code;
+        $authorizationHeader['Content-Type'] = 'application/json';
+
+        $response = HttpHelper::get($apiEndpoint, null, $authorizationHeader);
+
+        $user['user'] = [
+            'displayName'  => $response->users[0]->full_name,
+            'emailAddress' => $response->users[0]->email,
+        ];
```

Comment:
get only the logged user name and email, 

users[0] could be the guy who is not logged in.